src/handle_provide_parameter.c

```diff
@@ -107,18 +107,18 @@ static void handle_reward_create_position(ethPluginProvideParameter_t *msg, cont
 static void handle_lock_ids_array(ethPluginProvideParameter_t *msg, context_t *context) {
     switch (context->next_param) {
         case OFFSET:
-            context->next_param = LOCK_IDS_LEN;
+            context->next_param = LENGTH;
             break;
-        case LOCK_IDS_LEN:
+        case LENGTH:
```

Comment:
May be we can rename it to `ARRAY_LENGTH`?

Reply:
Done.